channels/mx.m3u

```diff
@@ -5,11 +5,21 @@ https://aztkhlslive3-i.akamaihd.net/hls/live/777407/adn40mx/master.m3u8
 https://aztkhlslive3-i.akamaihd.net/hls/live/777407/adn40mx/master03.m3u8
 #EXTINF:-1 tvg-id="AlcarriaTV.mx" tvg-name="Alcarria TV" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://i.imgur.com/9Xn7ZEZ.jpg" group-title="",Alcarria TV
 http://217.182.77.27/live/alcarriatv-livestream.m3u8
-#EXTINF:-1 tvg-id="XHJKTDT.mx" tvg-name="Azteca Uno" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://media.localbtv.com/images/sandiegobtv/logos/1.1.png" group-title="",Azteca Uno (XHJK-TDT) (1080p)
-https://cdnapisec.kaltura.com/p/2630451/sp/2630451/playManifest/entryId/1_83looakw/deliveryProfileId/672/protocol/https/format/applehttp/a.m3u8
-#EXTINF:-1 tvg-id="XHJKTDT.mx" tvg-name="Azteca Uno" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://media.localbtv.com/images/sandiegobtv/logos/1.1.png" group-title="",Azteca Uno (XHJK-TDT) (480p)
-https://cdnapisec.kaltura.com/p/2630451/sp/2630451/playManifest/entryId/1_jwma99nu/deliveryProfileId/672/protocol/https/format/applehttp/a.m3u8
-#EXTINF:-1 tvg-id="Canal10Cancun.mx" tvg-name="Canal 10 Cancún" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://i.imgur.com/vakdxny.jpg" group-title="",Canal 10 Cancún (720p)
+#EXTINF:-1 tvg-id="XHTITTDT.mx" tvg-name="Azteca 7" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://media.localbtv.com/images/sandiegobtv/logos/20551-91383.png" group-title="",Azteca 7 (XHTIT-TDT)
+https://streams.the6tv.duckdns.org:2443/locals/SanDiego/xhtit-tdt.m3u8
+#EXTINF:-1 tvg-id="XHJKTDT.mx" tvg-name="Azteca Uno" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://media.localbtv.com/images/sandiegobtv/logos/1.1.png" group-title="",Azteca Uno (XHJK-TDT)
+https://streams.the6tv.duckdns.org:2443/locals/SanDiego/xhjk-tdt.m3u8
+#EXTINF:-1 tvg-id="XHCTTITDT.mx" tvg-name="Imagen TV" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/1/1d/Logo_Imagen_Televisi%C3%B3n_2016.png/800px-Logo_Imagen_Televisi%C3%B3n_2016.png" group-title="",Imagen TV (XHCTTI-TDT)
+https://streams.the6tv.duckdns.org:2443/locals/SanDiego/xhctti-tdt.m3u8
+#EXTINF:-1 tvg-id="XHCTTITDT2.mx" tvg-name="Imagen Multicast" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://yt3.ggpht.com/ytc/AAUvwngCFclkx8_OjeImr3mO1BvDLtxIUkDcDL037PCAgw" group-title="",Imagen Multicast (XHCTTI-TDT)
```

Comment:
Keep the original call sign.

Reply:
So keep just XHCTTITDT instead of marking the DT2?

Reply:
Yes, that's how the channel is called.

Reply:
ok will change

Reply:
According to this : https://www.rabbitears.info/search.php?request=network_search&network=ImagenTV, it does.